bundle/compliance/cis_gcp/rules/cis_1_15/rule.rego

```diff
@@ -15,7 +15,7 @@ is_valid if {
 }
 
 finding = result if {
-	data_adapter.is_api_key
+	common.is_gcp_subtype("gcp-apikeys-key")
```

Comment:
the main difference here is that we don't introduce another layer of abstraction: `is_api_key` and instead define the string once in a set and use it whenever needed, this IMO makes it more explicit and less complicated